adam_core/coordinates/covariances.py

```diff
@@ -232,10 +225,124 @@ def sample_covariance(
     -------
     samples : `~numpy.ndarray` (num_samples, D)
         The drawn samples row-by-row.
+    W: `~numpy.ndarray` (num_samples)
+        Weights of the samples.
+    W_cov: `~numpy.ndarray` (num_samples)
+        Weights of the samples to reconstruct covariance matrix.
     """
     normal = multivariate_normal(mean=mean, cov=cov, allow_singular=True)
     samples = normal.rvs(num_samples)
-    return samples
+    W = np.full(num_samples, 1 / num_samples)
+    W_cov = np.full(num_samples, 1 / num_samples)
+    return samples, W, W_cov
+
+
+def sample_covariance_sigma_points(
+    mean: np.ndarray, cov: np.ndarray, alpha: float = 1, kappa: float = 0.0
+) -> Tuple[np.ndarray, np.ndarray, np.ndarray]:
+    """
+    Create sigma-point samples of a multivariate Gaussian distribution
+    with given mean and covariances.
+
+    Parameters
+    ----------
+    mean : `~numpy.ndarray` (D)
+        Multivariate mean of the Gaussian distribution.
+    cov : `~numpy.ndarray` (D, D)
+        Multivariate variance-covariance matrix of the Gaussian distribution.
+    alpha : float, optional
+        Spread of the sigma points between 1e^-2 and 1.
```

Comment:
Wann & Van Der Merwe 2000 suggests 1E-3, is that too low for us for some reason?

Reply:
Discussion about alpha values is in #41.